blocks/identity-block/features/header-account-action/default.jsx

```diff
@@ -49,6 +49,33 @@ const HeaderAccountAction = ({ customFields }) => {
     return () => { isActive = false; return null; };
   }, [Identity, loggedIn]);
 
+  useEffect(() => {
+    const closeMenuOnClick = (event) => {
+      if (event.target.closest('.menu')) {
+        return;
+      }
+      setAccountMenu(false);
+    };
+
+    window.addEventListener('click', closeMenuOnClick, true);
+    return () => {
+      window.removeEventListener('click', closeMenuOnClick);
+    };
+  }, []);
+
+  useEffect(() => {
+    const handleEscKey = (event) => {
+      if (event.keyCode === 27) {
+        setAccountMenu(false);
+      }
+    };
+
+    window.addEventListener('keydown', handleEscKey, true);
+    return () => {
+      window.removeEventListener('keydown', handleEscKey);
+    };
+  }, []);
```

Comment:
this looks like it could be helpful at some point. was this part of the ac? in the commit, it says it's for mobile?